cocos/spine/skeleton-system.ts

```diff
@@ -0,0 +1,69 @@
+import { director, Director } from '../core/director';
+import { System } from '../core/components';
+import { Skeleton } from './skeleton';
+
+export class SkeletonSystem extends System {
```

Comment:
Basically the same implementation, you can merge `SkeletonSystem` with `ArmatureSystem` to `SkeletonAnimation2D` (as we also have 3d skeleton animation). Reuse the same implementation

Reply:
Skeleton refers to Spine, Armature refers to Dragonbones, it's unreasonable to unify them as SkeletonAnimation. What's more, there are some slightly differences between their implementation. And the difference might be bigger and bigger in the future , although them are roughly similar now.

Reply:
You are right, separate them into spine and db module can also help to cull system code when module is desactivated.